doc/tutorials/static-ip.rst

```diff
@@ -21,8 +21,18 @@ on the target device and can also be found under ``tools/acrnbridge`` in the sou
 Setting up the static IP address
 ********************************
 
-You can set up a static IP address by modifying the
-``/usr/lib/systemd/network/50-eth.network`` file, and more specifically the ``[Network]``
+You can set up a static IP address by copying the
+``/usr/lib/systemd/network/50-eth.network`` file to
+``/etc/systemd/network/`` directory. You can create this directory and
+copy the file with the following command:
+
+.. code-block:: console
+
+   mkdir -p /etc/systemd/network
+   cp /usr/lib/systemd/network/50-eth.network /etc/systemd/network
+
+Then by modifying ``/usr/lib/systemd/network/50-eth.network``,
```

Comment:
Change this to:

    Modify the ``[Network]`` section in the ``/etc/systemd/network/50-eth.network``
    file you just created.

Reply:
Isn’t that rather the file in /etc that one should modify (i.e the copy)?

Reply:
Just noticed that @dbkinder caught that before me, thanks @dbkinder ! :-)